src/Questions.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from "react";
+const Question = (props) => {
+  const allAnswer = [
+    ...props.value.incorrect_answers,
+    props.value.correct_answer,
+  ];
+  const [selected, setSelected] = React.useState(null);
+  let styleQue = props.checkAns
```

Comment:
You should improve the readability of this code. Ternary operator is great when you have two options but deep nesting can be a bit confusing, in which case you should default to using `if..else if...else` or `switch` statement.